### PR TITLE
[tools] Split resources.json and add configure for suites

### PR DESCRIPTION
**Pull request by kangxu**
- Split resources.json into testharness.json, webrunner.json and xsl.json
- Add configure for apptools, behavior, cordova, embeddingapi, misc and stability
- Add option "-s" in copy.py to support update resources in specified test suite

### CODE REVIEW

File: tools/resources/resources.json [hunk 1]
```diff
@@ -128,7 +128,12 @@
         "webapi/webapi-vehicleinfo-ivi-tests/resources",
         "webapi/webapi-webcl-nonw3c-tests/resources",
         "webapi/webapi-webrtc-w3c-tests/resources",
-        "webapi/webapi-webspeech-w3c-tests/resources"
+        "webapi/webapi-webspeech-w3c-tests/resources",
```
Comment (zqzhang) [inline tools/resources/resources.json]: If possible, please make the list alphabetical, thanks.

File: tools/resources/resources.json [hunk 2]
```diff
@@ -276,7 +281,25 @@
         "wrt/wrt-securitymanu-tizen-tests/webrunner",
         "wrt/wrt-sharedmodemanu-android-tests/webrunner",
         "wrt/wrt-upgrademanu-android-tests/webrunner",
-        "wrt/wrt-uxmanu-android-tests/webrunner"
+        "wrt/wrt-uxmanu-android-tests/webrunner",
```
Comment (zqzhang) [inline tools/resources/resources.json]: Make this list alphabetical too.

File: tools/resources/resources.json [hunk 3]
```diff
@@ -449,6 +472,27 @@
         "wrt/wrt-packertool2-android-tests",
         "wrt/wrt-rt-linux-tests",
         "wrt/wrt-security-android-tests",
-        "wrt/wrt-sharedmode-android-tests"
+        "wrt/wrt-sharedmode-android-tests",
```
Comment (zqzhang) [inline tools/resources/resources.json]: Make this list alphabetical too.

File: tools/resources/resources.json (hunk 1 above)
Comment (zqzhang) [inline tools/resources/resources.json]: Don't forget the sub-app, e.g. #2418

Comment (haoxli) [inline tools/resources/resources.json]: @zqzhang There is too many sub-app, and some apps have packed the resources in app. If we maintain all sub app resource in copy.py, resource.json will be very large. I want to keep the resource.json only maintain test suites resources, can we copy them by manual to sub app if we need debug them?

Comment (zqzhang) [inline tools/resources/resources.json]: No, I don't want manually copy them on demand.

If resources.json is too long, you can split them, e.g. `testharness.json`, `webrunner.json`, etc.

**Comment by zqzhang** [comment]
Can we finish this today?

**Comment by kangxu** [comment]
@zqzhang  I have updated it following your comments

File: tools/build/copy.py [hunk 4]
```diff
@@ -30,12 +30,14 @@
...
 import os
+import os.path
+import glob
 import sys
 import shutil
 from optparse import OptionParser
 
 #
-# Usage: python copy.py  [-j /path/to/resources.json] [-n specify the file or dir you want to copy,please split them by comma]
+# Usage: python copy.py  [-j /path/to/resources.json]
```
Comment (zqzhang) [inline tools/build/copy.py]: Is this correct?

**Comment by zqzhang** [comment]
Good. Have you formatted the JSON files using `tools/format/json_formatter`?

**Comment by kangxu** [comment]
@zqzhang  I have formatted the JSON files using tools/format/json_formatter  and updated the usage descriptions of copy.py.

**Pull request merged**